itests/org.openhab.io.hueemulation.tests/itest.bndrun

```diff
@@ -3,7 +3,16 @@
 Bundle-SymbolicName: ${project.artifactId}
 Fragment-Host: org.openhab.io.hueemulation
 
--runrequires: bnd.identity;id='org.openhab.io.hueemulation.tests'
+-runrequires: \
+    bnd.identity;id='org.openhab.io.hueemulation.tests',\
+    bnd.identity;id='org.openhab.core.binding.xml',\
+    bnd.identity;id='org.openhab.core.thing.xml',\
+    bnd.identity;id='com.eclipsesource.jaxrs.jersey-all'
```

Comment:
That's a quick solution. In the mid and long run I do not want a direct dependency on that 5 years old buggy jax-rs bundle though :) (I mean I proposed that 3k diff core patch to get rid of that bundle, I'm that serious ;)

Reply:
Feel to propose another solution to get the integration test running. It was further away from running then I expected...

 I could imagine that we pick some changes from this PR as a first step forward to getting the build running.

Reply:
Of course